src/app.test.js

```diff
@@ -180,16 +201,31 @@ test("shows outstanding todos when filter is set to Outstanding", () => {
   expect(queryByDisplayValue(todoList, "Take dog for a walk!")).toBeVisible();
 });
 
-/**
- * SETUP
- * render todo app
- * input user name
- * add few todos
- * destroy todo app
- * ACT
- * re render todo app
- * input same user name
- * ASSERT
- * see previously added todos rendered
- */
-test.todo("renders todo list for existing user");
+test("renders todo list for existing user", () => {
```

Comment:
is this really what it is? :)

what I see here:

- user "A" logs in, adds 2 todos
- (a new) user "B" logs in (todo list should be emptied)
- user "A" logs back in and the belonging todos are loaded again

btw what does "local assert" mean here?